lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
Actually it does build in parallel by default. However, there's a -j option taking an integer which can be 1 for single-threaded builds. Sorry for not looking in this before submitting this pull request. I'll test and fix it.